subsystem/kubeapi_test.go

```diff
@@ -3277,6 +3277,48 @@ spec:
 		checkAgentClusterInstallCondition(ctx, installkey, hiveext.ClusterSpecSyncedCondition, hiveext.ClusterSyncedOkReason)
 		checkAgentClusterInstallConditionConsistency(ctx, installkey, hiveext.ClusterSpecSyncedCondition, hiveext.ClusterSyncedOkReason)
 	})
+
+	It("import installed cluster", func() {
+		By("deploy installed cluster deployment")
+		clusterDeploymentSpec.Installed = true
+		deployClusterDeploymentCRD(ctx, kubeClient, clusterDeploymentSpec)
+
+		By("deploy installed agent cluster install")
+		aci := &hiveext.AgentClusterInstall{
+			TypeMeta: metav1.TypeMeta{
+				Kind:       "AgentClusterInstall",
+				APIVersion: "hiveextension/v1beta1",
+			},
+			ObjectMeta: metav1.ObjectMeta{
+				Namespace: Options.Namespace,
+				Name:      clusterDeploymentSpec.ClusterInstallRef.Name,
+			},
+			Spec: *aciSpec,
+			Status: hiveext.AgentClusterInstallStatus{
+				Conditions: []hivev1.ClusterInstallCondition{
```

Comment:
Is it needed?
The conditions should be populated by controller, not by the user

Reply:
I agree that it doesn't make sense to ask from the user to set conditions but for now this is what we have in the code and it's out of scope for this ticket, i added this topic for our meeting tomorrow.
In order for us to create day2 cluster we need:
1. CRD: spec.Installed = ture, Spec.ProvisionRequirements.ControlPlaneAgents > 1
2. ACI complete conditions = true
The problem here is with 2, i think that if CD is marked as installed we can skip the validation of the conditions.

Reply:
Opened https://issues.redhat.com/browse/MGMT-8224 to handle it.

Reply:
Had a talk with Fred, looks like we don't really need to have the condition, fixed the tests.